test/page.spec.ts

```diff
@@ -1745,4 +1745,21 @@ describe('Page', function () {
       expect(page.browserContext()).toBe(context);
     });
   });
+
+  describe('WebVitals', () => {
+    it('should report all relevant metrics', async () => {
+      const { page, server } = getTestState();
+
+      const events = [];
+      await page.webvitals.on('*', (metric) => {
+        events.push(metric);
+      });
```

Comment:
Based on the example above, I was expecting `metric` to be an object with a `value` property, but the assertion on line 1762 suggests that it’s just a string?

Reply:
That is because of the '*' event listener. In that case, the event name is the first parameter. 
The object from the example would be the second argument.